### PR TITLE
swupdate: add swupdate info for intel NUC

### PR DESCRIPTION
**Pull request by TSAnsari**
Signed-off-by: Tariq Ansari <tansari@luxoft.com>

### CODE REVIEW

File: docs/chapters/architecture/SOTA-system.rst [hunk 1]
```diff
@@ -568,8 +572,12 @@ the machine running PELUX to let Hawkbit know our device exists.
 
 .. code-block:: bash
 
+    For raspberrypi3:
     $ swupdate -H raspberrypi3:1.0 -e stable,alt -f /etc/swupdate.cfg -l 5 -u '-t DEFAULT -u http://192.168.3.11:8080 -i DeviceID'
 
+    For intel-nuc:
```
Comment (sashko) [inline docs/chapters/architecture/SOTA-system.rst]: it's either the target name, either Intel NUC

**Review by FlorentRevest — APPROVED** [review]

File: docs/chapters/architecture/SOTA-system.rst [hunk 2]
```diff
@@ -568,8 +572,12 @@ the machine running PELUX to let Hawkbit know our device exists.
 
 .. code-block:: bash
 
+    For raspberrypi3:
```
Comment (sashko) [inline docs/chapters/architecture/SOTA-system.rst]: perhaps we should unify it? :)

Comment (TSAnsari) [reply docs/chapters/architecture/SOTA-system.rst]: Unification done. Please have a look

File: docs/chapters/architecture/SOTA-system.rst [hunk 3]
```diff
@@ -492,9 +492,9 @@ on an IP network:
   that this machine has the *192.168.3.11* IP address. This machine must have
   Java 8 (Both OpenJDK and Oracle Java 1.8 work), Maven and rabbitmq-server
   installed.
-- A raspberrypi3 running PELUX. Other platforms should have similar
-  configurations in the future but the Raspberry Pi is currently the only
-  supported platform.
+- A raspberrypi3 or an Intel NUC(intel-corei7-64) running PELUX. Other
```
Comment (sashko) [inline docs/chapters/architecture/SOTA-system.rst]: space before (

**Review by sashko — CHANGES_REQUESTED** [review]

**Comment by TSAnsari** [comment]
fixed the space

**Review by sashko — APPROVED** [review]

**Pull request merged**